Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/OwnershipSecurity.ts

```diff
@@ -5,7 +5,7 @@
 
 import { documentIdForDocumentInfo, FrontendRequest, Logger, writeRequestToLog } from '@edfi/meadowlark-core';
 import type { PoolClient, QueryResult } from 'pg';
-import { SecurityResult } from '../security/SecurityResponse';
+import { SecurityResult } from '../security/SecurityResult';
```

Comment:
Renamed SecurityResponse file to SecurityResult for consistency with type name in file.